core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -846,8 +904,15 @@ public FileSystemMasterView getFileSystemMasterView() {
   public List<AlluxioURI> checkConsistency(AlluxioURI path, CheckConsistencyOptions options)
       throws AccessControlException, FileDoesNotExistException, InvalidPathException, IOException {
     List<AlluxioURI> inconsistentUris = new ArrayList<>();
-    try (LockedInodePath parent = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
-      mPermissionChecker.checkPermission(Mode.Bits.READ, parent);
+    try (LockedInodePath parent = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ);
+         MasterAuditContext auditContext = createAuditContext("checkConsistency", path, null,
+             parent.getInode())) {
```

Comment:
In `checkConsistency` method, the expected behavior when the target inode does not exist is to throw a FileDoesNotExistException. I think the problem is who throws it. As discussed, auditContext should not throw exception.